server.js

```diff
@@ -3,19 +3,37 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+import netflixData from './data/netflix-titles.json'
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/shows"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
+const Show = mongoose.model('Show', {
+    show_id: String,
+    title: String,
+    director: String,
+    cast: String,
+    country: String,
+    date_added: String,
+    release_year: Number,
+    rating: String,
+    duration: String,
+    listed_in: String,
+    description: String,
+    type: String
+})
+
+const seedDatabase = async() => {
+    await Show.deleteMany()
+    netflixData.forEach(show => {
+
+        new Show(show).save()
+    })
+}
+
+seedDatabase()
```

Comment:
careful though, as this will reset and seed the database every time you start the server